bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 1.4.0
+version: 1.4.1
```

Comment:
I'm not sure whether to have this as a patch or bump the minor version instead. It is true that this is a fix for an existing functionality, but the changes may be extensive enough for a minor, WDYT?

Reply:
I think minor bump (as-is) since the changes focus solely on the setting `.Values.auth.tls.enabled` being true and getting it to the desired functionality. No new features have been introduced here.